src/components/Achievement/Achievement.test.js

```diff
@@ -1,16 +1,47 @@
 import React from 'react'
-import { shallow } from 'enzyme'
+import { render, screen } from '@testing-library/react'
+
+import FarmhandContext from '../../Farmhand.context'
 
 import Achievement from './Achievement'
 
-let component
+describe('<Achievement />', () => {
+  let achievementObject
 
-beforeEach(() => {
-  component = shallow(
-    <Achievement {...{ achievement: {}, completedAchievements: {} }} />
-  )
-})
+  beforeEach(() => {
+    achievementObject = {
+      description: 'the best achievement',
+      id: 'achievement-1',
+      name: 'achievement one',
+      rewardDescription: 'the greatest reward',
+    }
+
+    const farmhandContextValue = {
+      gameState: {},
+      handlers: {},
+    }
+
+    render(
+      <FarmhandContext.Provider value={farmhandContextValue}>
+        <Achievement
+          achievement={achievementObject}
+          completedAchievements={{}}
+        />
+      </FarmhandContext.Provider>
```

Comment:
This is a really smart pattern for testing components that require game state context. I'll use this for other test cases as well!